vivarium/__init__.py

```diff
@@ -16,3 +21,208 @@
 process_registry.register(DeriveCounts.name, DeriveCounts)
 process_registry.register(TimelineProcess.name, TimelineProcess)
 process_registry.register(NonSpatialEnvironment.name, NonSpatialEnvironment)
+
+
+# updater functions
+def update_merge(current_value, new_value, states):
```

Comment:
We could still define these in the registry file and import the updaters, serializers, etc. in `__init__.py`. This is mostly a stylistic preference of mine to put as little as possible in `__init__.py`